src/Components/Server/src/ComponentHub.cs

```diff
@@ -209,6 +209,17 @@ public async ValueTask EndInvokeJSFromDotNet(long asyncHandle, bool succeeded, s
             _ = circuitHost.EndInvokeJSFromDotNet(asyncHandle, succeeded, arguments);
         }
 
+        public async ValueTask SupplyByteArray(long id, byte[] data)
+        {
+            var circuitHost = await GetActiveCircuitAsync();
+            if (circuitHost == null)
+            {
+                return;
+            }
+
+            _ = circuitHost.SupplyByteArray(id, data);
```

Comment:
Any reason not to await this?

Reply:
We don't currently await the other interop methods (`BeginInvokeDotNetFromJS`, `EndInvokeJSFromDotNet`) so just following the same fire & forget convention here.

Reply:
I'm open to awaiting if you'd think it'd be beneficial?

Reply:
I guess if we're on an async code path, it makes sense to await it. It's not like the client expects this to be fire-and-forget, is it?